app/expences.py

```diff
@@ -0,0 +1,84 @@
+import math
+import datetime
+
+from app.customers import Customer
+from app.shops import Shop
+
+
+def find_min_cost(
+        customer: Customer, shops: list[Shop], fuel_price: float
+) -> tuple | None:
+    budget = customer.money
+    print(f"{customer.name} has {budget} dollars")
+    (shop_head_to,
+     total_cost,
+     purchase_cost_min,
+     _distance
+     ) = [shops[0], 0, 0, 0]
```

Comment:
In case you do not have any arguments on the same line with the closing parenthesis, your opening parenthesis should not have any arguments as well.